### ISSUE TITLE
Intermittent Freezing

### ISSUE THREAD

**Issue by Apexcery** [xivdev/Penumbra#291]
Ever since I bought a new super-ultrawide monitor (5120x1440), I've been getting intermittent freezing around every 5 minutes. When the game recovers I can see the disconnect icon next to my player name. It seems to occur more often when using abilities/opening menu's but can also happen when just standing around.

I've tried fully reinstalling FFXIV, XIVLauncher, and Penumbra, I've tried not using any plugins at all other than penumbra.

It doesn't matter which texture mod I have applied, as long as at least one mod is enabled, I'll get the freezing.

**Comment by Ottermandias** [comment]
Do you have dynamic resolution enabled?

**Comment by Apexcery** [comment]
I do not, should I have?

**Comment by Ottermandias** [comment]
No, but the SE settings for dynamic resolution and frame rate limiting are pretty bad and cause issues.

**Comment by Apexcery** [comment]
I have dynamic resolution turned off, and frame rate limit set to None
